collect_app/src/main/java/org/odk/collect/android/storage/migration/StorageMigrationDialog.java

```diff
@@ -0,0 +1,69 @@
+package org.odk.collect.android.storage.migration;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.DialogFragment;
+
+import org.jetbrains.annotations.NotNull;
+import org.odk.collect.android.R;
+
+public class StorageMigrationDialog extends DialogFragment {
```

Comment:
You should be able to use the `MaterialFullScreenDialogFragment` for this I think

Reply:
Fixed.